src/easystar.js

```diff
@@ -431,6 +449,23 @@ EasyStar.js = function() {
         return false;
     };
 
+    /**
+     * -1, -1 | 0, -1  | 1, -1
+     * -1,  0 | SOURCE | 1,  0
+     * -1,  1 | 0,  1  | 1,  1
+     */
+    var calculateDirection = function (diffX, diffY) {
+        if (diffX === 0, diffY === -1) return 'bottom'
+        else if (diffX === 1, diffY === -1) return 'bottom-left'
```

Comment:
This is the first time we're exposing directions to the API. I think it might make sense to make them constants on the EasyStar object.

```
EasyStar.BOTTOM = 'bottom'
EasyStar.TOP = 'top'
...
```